src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2535,6 +2576,22 @@ export class StudyViewPageStore
                         chartUniqueKey,
                         clinicalDataFilter
                     );
+                } else if (isGenePanelChart(chartUniqueKey)) {
+                    const filteredSampleIdentifiers = getFilteredSampleIdentifiers(
+                        this.samples.result.filter(sample =>
+                            values.includes(
```

Comment:
i wonder if we could improve the name ```values```.   i guess it's genePanelIds?

Reply:
`values` is a generic parameter for any type of chart. For my gene panel charts, it happens to be genePanelIds, but it can be any array of strings, so I think it's good to keep the generic name